src/js/localization.js

```diff
@@ -36,6 +36,28 @@ i18n.init = function(cb) {
                     }
             });
     });
+    // This function should do the same things that the i18n.localizePage function below does.
+    i18next.on('languageChanged', function (newLang) {
```

Comment:
I don't think this is the best way to do it. The code is very similar to function `localizePage`, I think is better to use the same function when is the first time that we localize the page or when we are doing it after each change of language. Maybe pass a parameter to a third method that can be used for both functions. In this way we will have only one place to maintain if needed.